drafts/use-cases/1.entry-point.md

```diff
@@ -20,24 +20,23 @@ in a configuration or in the input field.
 Once provided, the application is expected to connect to the entry-point of the Web API and download initial
 details on what else can be done with that API from that very point.
 
-```
+```http
 GET /api
 
 HTTP 200 OK
 ```
-```javascript
+```json
 {
     "@id": "/api",
-    "operations": [
+    "@type": "hydra:EntryPoint",
+    "operation": [
         {
-            "@id": "/api/events",
             "@type": "hydra:Operation",
-            "title": "Events",
+            "title": "List events",
             "method": "GET",
```

Comment:
This is a normal link. I wouldn't use operations for that.. and you certainly can't use them this way. This would say that if you do a `GET /api` you should expect to get a collection which contradicts what the you actually just got (an EntryPoint)